templates/cfn-abi-lacework-polygraph.template.yaml

```diff
@@ -698,6 +762,17 @@ Resources:
         LogAccountName: !Ref LogAccountName
         AuditAccountName: !Ref AuditAccountName
         KMSKeyIdentifierARN: !Ref KMSKeyIdentifierARN
+        AdministrationRoleARN: 
+          !If
+            - cControlTowerEnabled
+            - !Join [':', ['arn:aws:iam:', !Ref 'AWS::AccountId', 'role/service-role/AWSControlTowerStackSetRole']]
```

Comment:
for lines 768-769, do not hard-code partition, use AWS::Partition instead.

Also, consider simpler representation as stated below:
```
- !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/service-role/AWSControlTowerStackSetRole
- !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/${pAdminRoleName}

Reply:
Resolved in last commit.